lib/json/logger.ex

```diff
@@ -46,4 +45,8 @@ defmodule Json.Logger do
       end
     end
   end
+
+  def debug(message) do
```

Comment:
I don't think this will work for our objective which is to avoid all the `inspect` calls in the strings. The strings will still be passed "already calculated" into this function, and we and to avoid the calculation itself.

For it to work, I think you need to make `debug` a macro, that wraps the parameter in a lambda. Otherwise it will just be wrapping already calculated strings in the lambda.

Reply:
I removed debug function, didn't think about string concatenation but credo will now complain about 3rd level nesting in `array.ex` module last function.

Reply:
Meh, let credo complain. It's ok to ignore for this specific case.